app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java

```diff
@@ -141,6 +141,19 @@ public void onDataChange(@NonNull DataSnapshot snapshot) {
                         String newPlayersMissing = getString(R.string.lobby_player_missing,game.getMaxPlayerCount() - newPlayers.size());
 
                         playersMissing.setText(newPlayersMissing);
+                        GenericTypeIndicator<Boolean> coinIndicator = new GenericTypeIndicator<Boolean>() {
+                        };
+                        if(!game.getHost().equals(user)) {
+                            boolean started = snapshot.child("started").getValue(coinIndicator);
+                            if (started) {
+                                Intent intent = new Intent(getApplicationContext(), MapActivity.class);
+                                UserDatabaseProxy pdp = new UserDatabaseProxy();
```

Comment:
MapActivity expects a Player as an extra, not a User. This is one of the reasons tests are necessary, you would have caught this error.

Reply:
Done, thanks for pointing it out!